File: docs/src/pages/GuidePage/content/conversion/code/lcabyg/convert_lcabyg_results.rs
use lcax_convert::lcabyg;

fn main() {
    let file_path = Path::new("lcabyg_project.json");
    let project_content = fs::read_to_string(file_path).unwrap();
    let _file_path = Path::new("lcabyg_project_results.json");
    let result_content = fs::read_to_string(_file_path).unwrap();

    let project = lcabyg::parse::parse_lcabyg(
        &project_content,
        &result_content
    ).unwrap();
}

File: docs/src/pages/GuidePage/content/conversion/code/lcabyg/convertLCAbygResults.ts
import { convertLCAbyg } from 'lcax'
import fs from 'fs'

const lcabygData = fs.readFileSync('lcabyg_project.json')
const resultData = fs.readFileSync('lcabyg_project_results.json')
const project = convertLCAbyg(lcabygData, resultData)

// Output the LCAx formatted project to the console
console.log('LCAx', project)
